.github/workflows/ci.yml

```diff
@@ -27,18 +27,25 @@ jobs:
       CXX: clang++
       CC: clang
       PYTHON_BIN: /usr/bin/python3
+      # Install a specific version of Bazel that is supported by TensorFlow
+      # (see https://www.tensorflow.org/install/source#install_bazel).
+      BAZEL_VERSION: 1.1.0
     steps:
       - name: Installing bazel
         run: |
-          echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
-          sudo apt install curl
-          curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
-          sudo apt-get update && sudo apt-get install bazel
+          # https://docs.bazel.build/versions/master/install-ubuntu.html
+          sudo apt-get install unzip zip
+          wget https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+          chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+          ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh --user
```

Comment:
Huh looks like this was actually load bearing. Checking out the repo now fails because the directory isn't empty. So maybe go ahead and delete it. I guess we could also change everything to create a new directory for the repo, but if we're reconfiguring this we should probably switch to built-in actions anyway.